lib/src/main/java/graphql/nadel/engine/transform/result/NadelResultTransformer.kt

```diff
@@ -26,9 +33,23 @@ internal class NadelResultTransformer(private val executionBlueprint: NadelOvera
         service: Service,
         result: ServiceExecutionResult,
     ): ServiceExecutionResult {
+        if (result is NadelIncrementalServiceExecutionResult) {
```

Comment:
Can we remove this whole `if` block? It seems like transformations for incremental data is being done on the overloaded version of this function now.